src/Fulma/Common.fs

```diff
@@ -8,39 +8,26 @@ module Reflection =
     open Microsoft.FSharp.Reflection
     open System
 
-    #if FABLE_COMPILER
-    let inline getCaseName (case : 'T) =
-        Fable.Core.Reflection.getCaseName case
-    #else
     let getCaseName (case : 'T) =
+#if FABLE_COMPILER
```

Comment:
Can we please keep the inlined version when using Fable?

Reply:
I guess so, but inlining is not necessary for this function and I think it's better to share the signature when using conditional compilation to avoid mismatches. Why do you prefer to inline?

Reply:
Ah no, I though that by inlining the function he would be equivalent to call `Fable.Core.Reflection.getCaseName` directly but it will always create a "scope/function" and then call `Fable.Core.Reflection.getCaseName`. 

My bad